src/graphics/program-lib/chunks/bilateralDeNoise.frag

```diff
@@ -0,0 +1,103 @@
+// bilateral filter, based on https://www.shadertoy.com/view/4dfGDH# and
+// http://people.csail.mit.edu/sparis/bf_course/course_notes.pdf
+
+// A bilateral filter is a non-linear, edge-preserving, and noise-reducing smoothing filter for images.
+// It replaces the intensity of each pixel with a weighted average of intensity values from nearby pixels.
+// This weight can be based on a Gaussian distribution. Crucially, the weights depend not only on
+// Euclidean distance of pixels, but also on the radiometric differences (e.g., range differences, such
+// as color intensity, depth distance, etc.). This preserves sharp edges.
+
+#define SHADER_NAME BilateralDeNoise
```

Comment:
What's this for?

Reply:
it's an integration with SpectorJS .. you get the name in the capture list. Not essential, but nice.